internal/server/db/images.go

```diff
@@ -952,7 +952,11 @@ func (c *ClusterTx) CreateImage(ctx context.Context, project string, fp string,
 						JOIN projects_config AS t2 ON t2.project_id = profiles.project_id
 							AND t2.key = "features.profiles"
 							AND t2.value = "true"
-						WHERE profiles.name = "default"`, id)
+						WHERE profiles.name = "default"
+			UNION
+					SELECT ?, profiles.id FROM profiles
+						JOIN projects ON projects.id = profiles.project_id
+						WHERE projects.name = ? AND profiles.name = "default"`, id, id, imageProject)
```

Comment:
Logic seems good but looking at the existing code, we're not supporting our full range of true/false values. So we should probably pull the full data from `projects_config` so we can properly use IsTrue/IsFalse on the values, then once we know the relevant IDs we can perform the actual INSERT.

Reply:
Fixed.